docs/guide/search/extend_elasticsearch.md

```diff
@@ -524,7 +524,7 @@ For content-based aggregations, use the `ezplatform.search.elasticsearch.query.c
 
 !!! caution "Deprecated"
 
-    Search Facets are deprecated since version v3.2.
+    Search Facets are deprecated since version v3.2. Use Custom Aggregation instead
```

Comment:
```suggestion
    Search Facets are deprecated since version v3.2. Use a custom [Aggregation](../../api/public_php_api_search.md#aggregation) instead.
```